### ISSUE TITLE
Rails 6.1 support

### ISSUE THREAD

**Issue by AlecRust** [voormedia/rails-erd#363]
My ERB Rake task (the default generated by this gem) is running fine on Rails 6.0.3 but on Rails 6.1 I get:

```sh
Loading application environment...
Loading code in search of Active Record models...
Generating Entity-Relationship Diagram for 136 models...
rails aborted!
NoMethodError: undefined method `parent' for MyApplication::Application:Class
Did you mean?  present?
/Users/alec/projects/my-application/bin/rails:12:in `<top (required)>'
/Users/alec/projects/my-application/bin/spring:10:in `require'
/Users/alec/projects/my-application/bin/spring:10:in `block in <top (required)>'
/Users/alec/projects/my-application/bin/spring:7:in `tap'
/Users/alec/projects/my-application/bin/spring:7:in `<top (required)>'
Tasks: TOP => erd => erd:generate
(See full trace by running task with --trace)
```

**Comment by guapolo** [comment]
Hello, a possible cause is: domain.rb:52 won't work anymore (parent is deprecated on Rails 6.1 https://edgeguides.rubyonrails.org/6_1_release_notes.html#active-support-removals)

**Comment by guapolo** [comment]
I needed to finish my Rails version bump fast and continue working, so I just created a quick patch for this issue, at least it should provide a pointer to the maintainers of the project to solve this (I'm sure that they'll do it in a more elegant manner). Cheers 🍻

**Comment by guapolo** [comment]
Just saw that this is a duplicate of #358, better solution in that thread. Will close my PR.

**Comment by 0bserver07** [comment]
Ah thanks for the PR,  a quick hack here until someone responds:


```

mkdir lib/rails_erd/

touch lib/rails_erd/domain.rb

```

copy the commit content from this file [here](https://raw.githubusercontent.com/voormedia/rails-erd/cd225439688b6a9f7d74f28106fe2ada73090345/lib/rails_erd/domain.rb) to below file:

```
ib/rails_erd/domain.rb
```

**Comment by lulalala** [comment]
Can confirm the newly released [1.6.1](https://github.com/voormedia/rails-erd/pull/365#issuecomment-780089991) fixes this issue

**Comment by kerrizor** [comment]
@lulalala thanks for the confirmation!
/close
